tests/test_crs.py

```diff
@@ -146,3 +146,8 @@ def test_is_valid_false():
 
 def test_is_valid():
     assert is_valid_crs('EPSG:4326')
+
+
+def test_empty_json():
+    with pytest.raises(ValueError):
```

Comment:
Can you make this `CRSError`? https://github.com/mapbox/rasterio/blob/master/rasterio/errors.py#L6